go.mod

```diff
@@ -59,6 +60,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require github.com/holiman/uint256 v1.2.4 // indirect
```

Comment:
did you run tidy?

Reply:
Yes, I ran go mod tidy and no updates to go.mod and go.sum